Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/distance.h

```diff
@@ -160,56 +160,85 @@ double approximate_Hausdorff_distance_impl(
 }
 
 template<typename OutputIterator,
-         typename Geom_traits,
+         typename GeomTraits,
          typename NamedParameters,
          typename TriangleIterator,
          typename Randomizer,
          typename Creator,
          typename Derived>
-struct Triangle_structure_sampler_base{
+struct Triangle_structure_sampler_base
+{
   NamedParameters np;
-  Geom_traits geomtraits;
+  GeomTraits geomtraits;
   OutputIterator& out;
+
   Triangle_structure_sampler_base(OutputIterator& out,
                                   NamedParameters np)
     :np(np), out(out)
   {}
+
   void sample_points();
+
   double get_minimum_edge_length();
+
   template<typename Tr>
   double get_tr_area(const Tr&);
+
   template<typename Tr>
-  void get_tr_points(const Tr& tr, typename Geom_traits::Point_3 points[]);
-  void ms_edges_sample(std::size_t nb_points_per_edge,
-                       std::size_t nb_pts_l_u);
+  void get_tr_points(const Tr& tr, typename GeomTraits::Point_3 points[]);
+
+  void ms_edges_sample(const std::size_t& nb_points_per_edge,
+                       const std::size_t& nb_pts_l_u);
+
   void ru_edges_sample();
+
   Randomizer get_randomizer();
+
   void internal_sample_triangles(double, bool, bool);
+
   std::pair<TriangleIterator,TriangleIterator> get_range();
+
   std::size_t get_points_size();
+
   void procede()
   {
     using parameters::choose_parameter;
     using parameters::get_parameter;
     using parameters::is_default_parameter;
 
-    geomtraits = choose_parameter(get_parameter(np, internal_np::geom_traits), Geom_traits());
+    geomtraits = choose_parameter(get_parameter(np, internal_np::geom_traits), GeomTraits());
+
 
+    bool use_rs = choose_parameter
+        (get_parameter(np, internal_np::random_uniform_sampling), true);
+
+    bool use_gs = choose_parameter(
+          get_parameter(np, internal_np::grid_sampling), false);
 
-    bool use_rs = choose_parameter(get_parameter(np, internal_np::random_uniform_sampling), true);
-    bool use_gs = choose_parameter(get_parameter(np, internal_np::grid_sampling), false);
     bool use_ms = choose_parameter(get_parameter(np, internal_np::monte_carlo_sampling), false);
 
     if (use_gs || use_ms)
+    {
       if (is_default_parameter(get_parameter(np, internal_np::random_uniform_sampling)))
+      {
         use_rs=false;
+      }
+    }
+
+    bool smpl_vrtcs
```

Comment:
Good indentation is wrapping when it makes sense (not here), and grouping things together when they are similar to help the reader (here).

Reply:
@afabri insists on wrapping every if statement  even if it is not useful. I try to do it when I thik of it to please him, and avoid warnings or errors about misleading indent.

Reply:
I just wanted to write that.  And when you ask Laurent he will probably recommend to mimic the indentation style of the file.